sdks/java/core/src/main/java/org/apache/beam/sdk/io/Read.java

```diff
@@ -442,6 +443,19 @@ public void checkDone() throws IllegalStateException {
       public IsBounded isBounded() {
         return IsBounded.BOUNDED;
       }
+
+      @Override
+      public Progress getProgress() {
+        if (currentReader == null) {
+          return null;
```

Comment:
Noting that the return value is not `@Nullable`. Suggest re-enabling nullness checking here since it would have caught this. (you could perhaps suppress it for other parts of the file you don't want to fix.

Reply:
We've replaced with Progress.NONE as a placeholder for this (0 work done, 1 remaining)